api-tests/specs/practitioner-service/createperson.js

```diff
@@ -1,47 +1,47 @@
 const requests = require('../../data/practitioner-service/createperson')
 const conf = require('../../config/conf')
-
+const endpointUri = ':80/roles';
```

Comment:
note sure about this one - person test file and roles endpoint?

Reply:
Sure corrected it, thanks @tamirkamara